src/herder/test/UpgradesTests.cpp

```diff
@@ -214,6 +214,27 @@ toUpgradeType(LedgerUpgrade const& upgrade)
     return result;
 }
 
+LedgerHeader
+executeUpgrades(Application& app, xdr::xvector<UpgradeType, 6> const& upgrades)
```

Comment:
we should put all those helper functions in the empty namespace as they're only used in this file